### PR TITLE
Added section for minimum parameters (and a prerequisite note) when importing EKS clusters in 2.5, 2.6

### LINKED ISSUE #3791: [Rancher2] Clarify what parameters should be defined when importing EKS clusters (or not)
Node Groups and other parameters should not be defined when importing EKS clusters:

"Nothing needs to be defined beyond some small minimum set of parameters and Rancher will import the rest of the information from EKS. This should be outlined in the docs (and it was, but in a refactor, Donnie's addition was removed)."

https://jira.suse.com/browse/SURE-3882?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel

### PR DESCRIPTION
**Pull request by jtravee**
Closes #3791 

References: 
https://jira.suse.com/browse/SURE-3882?page=com.atlassian.jira.plugin.system.issuetabpanels%3Aall-tabpanel
https://rancher.com/docs/rancher/v2.6/en/cluster-provisioning/registered-clusters
https://rancher.com/docs/rancher/v2.5/en/cluster-provisioning/registered-clusters

### CODE REVIEW

File: content/rancher/v2.5/en/cluster-provisioning/registered-clusters/_index.md [hunk 1]
```diff
@@ -110,6 +119,32 @@ The option can also be specified using the environment variable `K3S_KUBECONFIG_
 $ curl -sfL https://get.k3s.io | K3S_KUBECONFIG_MODE="644" sh -s -
 ```
 
+### Configuring an Imported EKS Cluster with Terraform
+
+You should define **only** the minimum fields that Rancher requires when importing an EKS cluster with Terraform. This is important as Rancher will overwrite what was in the EKS cluster with any config that the user has provided, whether that config is valid or not.
```
Comment (thedadams) [inline content/rancher/v2.5/en/cluster-provisioning/registered-clusters/_index.md:124]: This makes it sound like our EKS implementation doesn't check that the user-provided config is a valid EKS config. There are some checks and EKS won't allow Rancher to set a completely invalid config.

Of course, you'll know the best wording to use, but something like this:
```suggestion
You should define **only** the minimum fields that Rancher requires when importing an EKS cluster with Terraform. This is important as Rancher will overwrite what was in the EKS cluster with any config that the user has provided. Even a small difference between the current EKS cluster and a provided user config could have unexpected results.
```

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md [hunk 2]
```diff
@@ -82,6 +86,32 @@ The option can also be specified using the environment variable `K3S_KUBECONFIG_
 $ curl -sfL https://get.k3s.io | K3S_KUBECONFIG_MODE="644" sh -s -
 ```
 
+### Configuring an Imported EKS Cluster with Terraform
+
+You should define **only** the minimum fields that Rancher requires when importing an EKS cluster with Terraform. This is important as Rancher will overwrite what was in the EKS cluster with any config that the user has provided, whether that config is valid or not.
```
Comment (thedadams) [inline content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md:91]: Same comment as above.

**Review by thedadams — CHANGES_REQUESTED** [review]

File: content/rancher/v2.5/en/cluster-provisioning/registered-clusters/_index.md (hunk 1 above)
Comment (jtravee) [reply content/rancher/v2.5/en/cluster-provisioning/registered-clusters/_index.md]: Updated as suggested; added a warning note to be more noticeable as well.

File: content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md (hunk 2 above)
Comment (jtravee) [reply content/rancher/v2.6/en/cluster-provisioning/registered-clusters/_index.md]: Updated as suggested; added a warning note to be more noticeable.

**Review by thedadams — APPROVED** [review]

**Pull request merged**